examples/soundspeed/server/ex_server_sis5.py

```diff
@@ -0,0 +1,31 @@
+import time
+import logging
+
+from hyo2.soundspeedmanager import app_info
+from hyo2.soundspeed.soundspeed import SoundSpeedLibrary
+from hyo2.soundspeed.server.server import Server
+from hyo2.abc.lib.logging import set_logging
+
+ns_list = ["hyo2.soundspeed", "hyo2.soundspeedmanager", "hyo2.soundspeedsettings"]
+set_logging(ns_list=ns_list)
+
+logger = logging.getLogger(__name__)
+
+lib = SoundSpeedLibrary()
+lib.listen_sis5()
+time.sleep(3)  # for SIS emulator
+lib.setup.server_echosounder_id = "EM124_101"

```

Comment:
We should retrieve the Echosounder ID by listening the datagrams. It would avoid an endless source of problems in our users.